src/main/java/calculatorFunction/Calculator.java

```diff
@@ -1,62 +1,73 @@
 package calculatorFunction;
 
-import java.util.List;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
 import java.util.ArrayList;
+import java.util.HashMap;
 
 public class Calculator {
 
-    public static void execute() {
+    private ArrayList<String> parsedValues = null;
```

Comment:
```suggestion
    private ArrayList<String> parsedValues;
```
생성자에서 초기화를 하기 때문에 null 로 명시적으로 초기화는 필요 없을 것 같아요 :)